backend/app/store/service/service.go

```diff
@@ -154,12 +162,6 @@ func (s *DataStore) Get(locator store.Locator, commentID string, user store.User
 	return s.alterComment(c, user), nil
 }
 
-// Put updates comment, mutable parts only
```

Comment:
Please, return it. For context, please see #1528.

Reply:
https://github.com/umputun/remark42/pull/757/commits/3e4515b0cbecc0927cfc3a47defbee1f3dd3675e#diff-be11453fbc15fb371635d660cf48b74ef5c375e7ad0368e26a94ac4ba431e67bR165